src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -24,13 +24,30 @@ const ReworkRequest = () => {
 
   const updateHandler = () => {
     if (request.data && currentStage?.previous) {
-      const newStage = currentStage.previous;
-      const eventTitle = currentStage.previousEventTitle;
-      updateStage.mutate({
+      let newData = {
         requestId,
-        newStage,
-        eventTitle,
-      });
+        newStage: "",
+        newSubStage: "",
+        eventTitle: "",
+      };
+
+      const subStage = currentStage.subStages?.find(
+        ({ key }) => key === request.data.subStage
+      );
+
+      if (subStage && subStage.previous) {
```

Comment:
Same comment as the SendRework.  Should be okay since should be no Sub Stages in DRAFT, so we should probably remove subStages from the STARTSTAGE type.  Or rework logic here to allow for subStage in first stage.